test/e2e/core/ArrowKeyNavContextMenu.test.js

```diff
@@ -16,8 +16,8 @@ test.describe('Using arrow keys to navigate context menu', () => {
 
   test('Testing layer contextmenu', async () => {
     await page.waitForTimeout(500);
-    await page.click('body > mapml-viewer');
```

Comment:
this is [not recommended by playwright](https://playwright.dev/docs/next/api/class-page#page-click), so I changed a few of them where they seemed to cause some flakiness